internal/manifests/api/interface.go

```diff
@@ -0,0 +1,22 @@
+package api
+
+import (
+	"context"
+
+	"github.com/openshift/assisted-service/models"
+	"github.com/openshift/assisted-service/restapi"
+	operations "github.com/openshift/assisted-service/restapi/operations/manifests"
+)
+
+//go:generate mockgen -package api -destination mock_manifests_api.go . ManifestsAPI
+type ManifestsAPI interface {
+	restapi.ManifestsAPI
+	ClusterManifestsInternals
```

Comment:
Why don't we just use `CreateClusterManifestInternal`, `ListClusterManifestsInternal` and `DeleteClusterManifestInternal` directly here?